### ISSUE TITLE
[instagram-feed] Summary of the issue

### ISSUE THREAD

**Issue by ozfiddler** [getgrav/grav-premium-issues#281]
v1.0.0

I am finding that after 3 months the Instagram feed disappears. I have to manually refresh the token. Scheduler is set up and enabled. I suspect the problem is that for the scheduler to work you need to be actually logged into your Instagram account (?). When I click on refresh token it asks me to login, then I am able to select "allow" and it is all working again.

I work with a number of clients with Instagram and I never know who I will be logged in as, at any given time. I'm sure there will be other devs who want to use this plugin on multiple sites with multiple Instagram accounts.

**Comment by MattAppleton** [comment]
Same here — I currently see an 'invalid token' message... and am unable to login to instagram form the plugin admin, I see this message:

`{"error_type": "OAuthException", "code": 400, "error_message": "Invalid scope: []"}`

**Comment by MattAppleton** [comment]
If I login to the relevant account it does successfully refresh if I hit the admin 'login to instagram' button again...

**Comment by FabriceJoseph** [comment]
I have the same issue : {"error_type": "OAuthException", "code": 400, "error_message": "Invalid scope: []"}.

**Comment by rhukster** [comment]
Ok will investigate this further. an invalid scope seems to indicate something with oauth has changed.

**Comment by ozfiddler** [comment]
No, I have no trailing slash.
![url](https://user-images.githubusercontent.com/104406200/174688150-173303ac-2920-4c27-b6cc-4cf185738b87.jpg)

**Comment by w00fz** [comment]
Can you try adding it and see if that resolves the issue?

**Comment by ozfiddler** [comment]
OK. I think it is due to refresh some time next month so will see what happens. There is no other way to test is there?

**Comment by w00fz** [comment]
There is a CLI `bin/plugin instagram-feed refresh-token` you might be able to try.

**Comment by ozfiddler** [comment]
No, don't have access to command line. Just realised it is 60 day expiry (I thought it was 3 months) so it should be coming up soon. I posted on 26 April. Will see how it goes.

**Comment by MattAppleton** [comment]
Still not working here — the token itself seems to be good, but the template no longer loads my feed... I am trying to call it like so:

```
<div id="instagram-feed" class="border-l border-t flex flex-row flex-wrap justify-center"></div>
```

I get a very empty div! Example here on this page: https://www.tarmachancafe.com

**Comment by MattAppleton** [comment]
It's working again now! 

Template is fine, it was the auth process that was bunging things up — logging back into Facebook developer there was something called 'Data use Checkup' that I needed to do...?!

God I hate Facebook.

**Comment by MattAppleton** [comment]
I DONT have a trailing slash by the way...

**Comment by rhukster** [comment]
Closed due to inactivity
